include/mxnet/lib_api.h

```diff
@@ -229,20 +241,60 @@ enum MXReturnValue {
   MX_SUCCESS = 1,
 };
 
+// For sparse input, read/write the data from NDarray via pointers.
```

Comment:
can we fix the comment now that MXSparse is for input and output? Maybe just say "sparse tensors"

Reply:
Updated it.